### PR TITLE
Update test/e2e/framework/volume owners file

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the test/e2e/framework/volume owners file.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/volume/OWNERS [hunk 1]
```diff
@@ -7,6 +7,7 @@ approvers:
 - jingxu97
 - jsafrane
 - msau42
+- xing-yang
```
Comment (msau42) [inline test/e2e/framework/volume/OWNERS:10]: maybe we should use the sig-storage owners aliases?

Comment (xing-yang) [reply test/e2e/framework/volume/OWNERS:10]: The names here are not exactly the same as the sig-storage-approvers owners aliases though: https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES#L173

The sig-storage-reviewers list differ even more.  I don't want to drop anyone's name here.

Comment (msau42) [reply test/e2e/framework/volume/OWNERS:10]: Can we specify aliases + diffs?

Comment (xing-yang) [reply test/e2e/framework/volume/OWNERS:10]: For approvers, I think it is fine.  For reviewers, there are quite a few names in sig-storage-reviewers that are not in this reviewer list.

Comment (msau42) [reply test/e2e/framework/volume/OWNERS:10]: Yes, I meant can you specify both the alias and the additional names in the same list? Like:
```
reviewers:
  sig-storage-reviewers
  abc
  xyz
```

Comment (xing-yang) [reply test/e2e/framework/volume/OWNERS]: Done

**Pull request merged**